packages/components/package.json

```diff
@@ -40,12 +41,14 @@
     "react": ">= 16.8.0"
   },
   "dependencies": {
-    "@chanzuckerberg/czedi-kit-styles": "^0.0.1-alpha.5",
```

Comment:
I kept hitting a build error where it couldn't find this package 😭  that's why I removed lines from `.storybook/preview.tsx`, and why there are Percy changes

I don't know if it's an easy fix, but soon we won't need this dependency anyways (and the button is more of a playground component) so I felt ok about it

Reply:
locally, or in CI? This should be fixed by a `npx lerna bootstrap`

Reply:
hm I saw it on CI but can try doing those and pushing up again

Reply:
ah wild, running that locally fixed it on CI also 👍